tests/api-mongo.test.js

```diff
@@ -466,6 +466,39 @@ describe('MongoDB Words', () => {
         });
     });
 
+    it('should return a list of igbo terms when using english by using single qoutes', (done) => {
+      const keyword = '\'water\'';
+      getWords({ keyword })
+        .end((_, res) => {
+          expect(res.status).to.be.equal(200);
+          expect(res.body).to.be.an('array');
+          expect(res.body).to.have.lengthOf.at.least(1);
+          expect(every(res.body, (word) => word.includes('water'))).to.equal(true);
+          done();
+        });
+    });
+
+    it('should also return a list of igbo terms when using english by using double qoutes', (done) => {
+      const keyword = '"water"';
+      getWords({ keyword })
+        .end((_, res) => {
+          expect(res.status).to.be.equal(200);
+          expect(res.body).to.be.an('array');
+          expect(res.body).to.have.lengthOf.at.least(1);
+          expect(every(res.body, (word) => word.includes('water'))).to.equal(true);
+          done();
+        });
+    });
+
+    it('should not return any words when wrapping an igbo word in quotes', (done) => {
+      const keyword = '"mmili"';
+      getWords({ keyword })
+        .end((_, res) => {
+          expect(res.status).to.be.equal(400);
```

Comment:
The response shouldn't be `400` since it's not an error to return an empty list of words. Rather, you should expect a status code of `200` and the `res.body` to have a length of zero